### PR TITLE
Cleanup: improve metav1 package import statement

### PR DESCRIPTION
**Pull request by my-git9**
Improve metav1 package import statement

### CODE REVIEW

**Comment by gandro** [comment]
Thanks for the PR! It seems there are some build failures, could you please address them:

https://github.com/cilium/cilium/actions/runs/3986769699/jobs/6835733649#step:9:134

File: pkg/k8s/slim/k8s/client/clientset/versioned/typed/discovery/v1beta1/endpointslice.go [hunk 1]
```diff
@@ -7,11 +7,12 @@ package v1beta1
 
 import (
 	"context"
+	v1 "k8s.io/client-go/applyconfigurations/meta/v1"
```
Comment (gandro) [inline pkg/k8s/slim/k8s/client/clientset/versioned/typed/discovery/v1beta1/endpointslice.go]: These files are auto-generated, as the header states. Please do not edit them.

**Review by gandro — CHANGES_REQUESTED** [review]

**Review by gandro — APPROVED** [review]

**Review by christarazi — CHANGES_REQUESTED** [review]
Thanks for the PR. In the commit body, can you say "This commit contains no functional changes."?

**Comment by my-git9** [comment]
> Thanks for the PR. In the commit body, can you say "This commit contains no functional changes."?

Updated. Thanks ~

**Review by christarazi — APPROVED** [review]

**Comment by christarazi** [comment]
Marking ready to merge as we have approving reviews and CI has passed except for known flakes.

**Pull request merged**